docs/one-click-integrations/bitbucket-integration.md

```diff
@@ -0,0 +1,163 @@
+# Bitbucket integration
+
+!!! note
+    <!--NOTE Limitation from Bitbucket integration V1-V2-->Pulse's Bitbucket integration is new and still in Beta:
+
+    -   For now, we only show the [Accelerate metrics](../metrics/accelerate.md) using your merged pull requests to automatically detect deployments.
+    -   <!--NOTE Limitation from Bitbucket integration V1-->At the moment, we're only collecting new data, so you won’t see any metrics after completing the integration setup. We'll automatically import historical data and backfill your charts within a few weeks.
+
+<!--NOTE Limitation from Bitbucket integration V1-V2-->Pulse integrates with Bitbucket Cloud to receive data about changes and deployments, necessary to calculate the metrics:
+
+-   [Lead time for changes](../metrics/accelerate.md#lead-time-for-changes)
+
+-   [Deployment frequency](../metrics/accelerate.md#deployment-frequency)
+
+## Setting up the Bitbucket integration
+
+!!! note
+    -   The Bitbucket integration connects to a specific [workspace](https://support.atlassian.com/bitbucket-cloud/docs/what-is-a-workspace/). You must be the **owner** of that Bitbucket workspace to proceed with the integration setup.
+    -   [See below](#bb-permissions) the permissions that Pulse requires from your Bitbucket account.  
+
+To set up the Bitbucket integration:
+
+1.  On Pulse, [expand **Integrations** and select **Bitbucket**](https://app.pulse.codacy.com/integrations/bitbucket){: target="_blank"}.
+
+1.  Click **Install Bitbucket App** and follow the instructions on the Bitbucket UI to install the app on your account.
+
+    ![Installing the Pulse Bitbucket App](images/bb-installing.png)
+
+1.  Wait until you get a confirmation that Pulse successfully connected the integration to Bitbucket.
+
+    ![Pulse Bitbucket integration set up successfully](images/bb-ok.png)
+
+    If there was an error please [contact support](mailto:pulsesupport@codacy.com).
+
+1.  Choose the workspace that you want to connect the integration. You will only be able to select a workspace that you are the owner.
```

Comment:
A fragment is missing here:
```suggestion
1.  Choose the workspace that you want to connect the integration to. You will only be able to select a workspace that you are the owner.
```
Although it reads a bit clunky. We may also want to consider: `Choose the workspace that you want to connect to.` or `Choose the workspace that you want to integrate with.`, depending on which one makes more sense.

Reply:
I prefer to keep the "owner" to match Bitbucket terminology. Also, I'll use "connect the integration to", to be consistent with the UI (on the UI we have "To connect to a specific workspace you must have owner permissions").

So, to keep the sentence consistent with the UI, I'll go with:
"Choose the workspace that you want to connect the integration to. You can only select a workspace where you have owner permissions."
(I'll do the update manually.)

Reply:
Updated!